components/supervisor/frontend/src/index.ts

```diff
@@ -30,6 +30,12 @@ window.addEventListener('error', (event) => {
             labels['error'] = 'LoadError';
         }
     }
+    if (event.error) {
+        (async () => {
+            const gitpodServiceClient = await pendingGitpodServiceClient;
```

Comment:
It seems to be failure point if this method breaks itself. Could we avoid waiting and only provide info which we have at the current point of time, i.e. get workapceId from Url, and instanceid, userid later?